disnake/emoji.py

```diff
@@ -231,13 +235,15 @@ async def delete(self, *, reason: Optional[str] = None) -> None:
         Deletes the custom emoji.
 
         You must have :attr:`~Permissions.manage_guild_expressions` permission to
-        do this.
+        do this, if this is not an app emoji.
```

Comment:
If this is not an app emoji, you must have ....

Reply:
https://github.com/DisnakeDev/disnake/pull/1388/commits/8e8e351defca90927b569e3826352ad8c5376261